### PR TITLE
Fixes #10673: Changes needed to upgrade to Pulp 2.6.2

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: test/glue/pulp/consumer_test.rb [hunk 1]
```diff
@@ -184,6 +184,10 @@ def test_uninstall_package_group
 
     def test_install_consumer_errata
       erratum_id = RepositorySupport.repo.errata_json.select { |errata| errata['id'] == 'RHEA-2010:0002' }.first['id']
+      profile = [{"vendor" => "FedoraHosted", "name" => "elephant",
+                  "version" => "0.3", "release" => "0.8",
+                  "arch" => "noarch", :epoch => ""}]
```
Comment (jlsherrill) [inline test/glue/pulp/consumer_test.rb]: i remember you saying that the lack of a package profile was causing pulp to error when performing this action, was an upstream issue opened for this?

Comment (ehelms) [inline test/glue/pulp/consumer_test.rb]: Upstream meaning what exactly?

Comment (jlsherrill) [inline test/glue/pulp/consumer_test.rb]: upstream pulp

Comment (ehelms) [inline test/glue/pulp/consumer_test.rb]: Pulp started returning a 400 bad response if there was no package profile present which isn't really an error (but manifests so on our side). So I am not sure that anything needs to be filed?

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**